rust/azure_iot_operations_protocol/src/telemetry/sender.rs

```diff
@@ -189,6 +189,9 @@ pub struct Message<T: PayloadSerialize> {
     /// Cloud event of the telemetry message.
     #[builder(default = "None")]
     cloud_event: Option<CloudEvent>,
+    /// Indicates whether the message should be retained or not.A retained message is a normal MQTT message with the retained flag set to true.
+    #[builder(default = "false")]
+    retain: bool,
```

Comment:
Is this only relevant to the `Sender`? Might there be any reason the `Receiver` (or rather, the application that's using it) could also care about this?

Reply:
Didn't see any place in receiver that it might be needed

Reply:
Might be worth changing the comment to something like "retained by the broker" to clarify that this affects MQ rather than the receiver.